README.md

```diff
@@ -200,11 +266,143 @@ df <- read.df(table_path, "deltaSharing")
 
 You can try this by running our [examples](examples/README.md) with the open, example Delta Sharing Server.
 
+### CDF 
+Starting from release 0.5.0, querying [Change Data Feed](https://docs.databricks.com/delta/delta-change-data-feed.html) is supported with delta sharing.
+Once the provider turn on CDF on the original delta table and share it through delta sharing. Once CDF is shared, the recipient can query
```

Comment:
The first sentence seems like it is missing something. So what's next after provider turns on CDF?